kombu/utils/limits.py

```diff
@@ -74,10 +62,10 @@ def expected_time(self, tokens=1):
         tokens = max(tokens, _tokens)
         return (tokens - _tokens) / self.fill_rate
 
-    def _get_tokens(self):
+    def _get_tokens(self) -> int:
         if self._tokens < self.capacity:
             now = monotonic()
             delta = self.fill_rate * (now - self.timestamp)
             self._tokens = min(self.capacity, self._tokens + delta)
             self.timestamp = now
-        return self._tokens
+        return int(self._tokens)
```

Comment:
From where are we inferring it should be `int`?

Reply:
This looks like an oversight on my part. Both the `tokens` args in the `expected_time` method and the return type of `_get_tokens` look like they should be floats.

Updated in 7589dc54